src/nrc/api/viewsets.py

```diff
@@ -56,6 +59,7 @@ class SubscriptionViewSet(
         "create": SCOPE_EVENTS_CONSUMEREN,
         "destroy": SCOPE_EVENTS_CONSUMEREN,
         "update": SCOPE_EVENTS_CONSUMEREN,
+        "partial-update": SCOPE_EVENTS_CONSUMEREN,
```

Comment:
Ik zie dat de scopes niet echt consistent gedefinieerd hier staan maar volgens mij kun je op twee manieren tegen de scopes aankijken:

1. Voor alle acties wat betreft een `Subscription` heb je alleen scope `SCOPE_EVENTS_CONSUMEREN` nodig; Met `Subscription`'s publiseer je technisch gezien niet een `Event` maar abboneer je jezelf op bepaalde `Event`'s en ben je eigenlijk alleen `Event`'s aan het lezen.
2. We trekken het begrip `SCOPE_EVENTS_PUBLICEREN` breeder dan alleen `Event`'s en passen dit ook toe op de andere resources. Dit betekent dat alle `create`/`update`/`partial-update`/`destroy` acties bij de andere resources (`Subscription` en `Domain`) de scope `SCOPE_EVENTS_PUBLICEREN` nodig hebben.

Wat vind jij logisch om toe te passen?